### DISCUSSION TITLE
/restart-github-actions-workflow-failed/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#556] [Comments]
Restart your GitHub Actions workflow when something failed
        
[https://www.eliostruyf.com/restart-github-actions-workflow-failed/](https://www.eliostruyf.com/restart-github-actions-workflow-failed/)

**Comment by netryxdev** [comment]
how can i get or create the secrets.ACTIONS_PAT ?

**Reply by estruyf** [reply]
You can get your own Personal Access Token (PAT) by following the next guide from GitHub: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic
